### PR TITLE
Update address

### PR DESCRIPTION
**Pull request by MadayAlcala**
Last changes of the project.

### CODE REVIEW

File: src/main/java/AutomationPractice/ui/pages/LoginPage.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
     /**
      * This method is in charge of the login to the application.
      *
-     * @param email 'email', it indicates the email with which the login will be carried out.
+     * @param email    'email', it indicates the email with which the login will be carried out.
```
Comment (TheYisusGithub) [inline src/main/java/AutomationPractice/ui/pages/LoginPage.java]: Please remove the space in blank between "email    'email'"

File: src/main/java/core/selenium/WebDriverFactory.java [hunk 2]
```diff
@@ -3,17 +3,29 @@
...
 /**
  * WebDriverFactory class, this class initializes a browser according to the type it receives.
  *
  * @author Maday Alcala
  * @version 0.0.1
  */
 public final class WebDriverFactory {
-    private static WebDriver webDriver;
-    private static final String MESSAGE_FOR_UNKNOWN_BROWSER = "Unknown browser type";
+    private static Map<BrowserType, IBrowser> map;
```
Comment (TheYisusGithub) [inline src/main/java/core/selenium/WebDriverFactory.java]: Please you most be become variable of method.

File: src/test/java/steps/LoginSteps.java [hunk 3]
```diff
@@ -16,13 +17,27 @@
  */
 public class LoginSteps {
     private LoginPage loginPage;
+    private HomePage homePage;
+
+
```
Comment (TheYisusGithub) [inline src/test/java/steps/LoginSteps.java]: Please remove this space in blank.

File: src/main/java/core/selenium/WebDriverFactory.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/core/selenium/WebDriverFactory.java]: Solved.

File: src/test/java/steps/LoginSteps.java (hunk 3 above)
Comment (MadayAlcala) [reply src/test/java/steps/LoginSteps.java]: Solved.

**Review by TheYisusGithub — APPROVED** [review]

**Pull request merged**